src/gstreamer/setup.py

```diff
@@ -1,27 +1,31 @@
+import os
+from glob import glob
 from setuptools import find_packages, setup
 
-package_name = 'gstreamer'
+package_name = "gstreamer"
 
 setup(
     name=package_name,
-    version='0.0.0',
-    packages=find_packages(exclude=['test']),
+    version="0.0.0",
+    packages=find_packages(exclude=["test"]),
     data_files=[
-        ('share/ament_index/resource_index/packages',
-            ['resource/' + package_name]),
```

Comment:
I swear the lack of linter standardization gives me a headache whenever I look at these

Reply:
Kinda optional though

Reply:
It's because I might be the only one with a python autoformatter setup lol, I definitely want to look into enforcing linting over the summer with a github actions workflow or something